File: ASMR.Web/ClientApp/src/data/release/common/iOSReleaseInformation.ts
import AppReleaseInformation from "@asmr/data/release/common/AppReleaseInformation"
import StoreReleaseInformation from "@asmr/data/release/common/StoreReleaseInformation"

interface iOSReleaseInformation extends AppReleaseInformation {
	AppStore: StoreReleaseInformation
}

export default iOSReleaseInformation

File: ASMR.Web/ReleaseInformation/Common/iOSReleaseInformation.cs
//
// asmr: Coffee Beans Management Solution
// © 2021 Pandora Karya Digital. All right reserved.
//
// Written by Danang Galuh Tegar Prasetyo [connect@danang.id]
//
// iOSReleaseInformation.cs
//

using System.Text.Json.Serialization;

// ReSharper disable InconsistentNaming
namespace ASMR.Web.ReleaseInformation.Common;

public class iOSReleaseInformation : AppReleaseInformation
{
	[JsonPropertyName("AppStore")] public StoreReleaseInformation AppStore { get; set; }
}